### DISCUSSION TITLE
Vitest run coverage fails after adding Storybook

### DISCUSSION THREAD

**Discussion by hfall** [storybookjs/storybook#25948] [Help]
### Summary

Our team's project (set up in vite), uses Vitest with testing library. The command `vitest run --coverage` works as expected. Then, we decided to use Storybook for many of its benefits, however we wanted to stick with our original unit tests without using stories (at least for now). When we run the coverage test, the test runner hangs forever and eventually fails.

Storybook framework is `@storybook/react-vite`

Is there some conflicting configs or packages that cause this?

Our `setupTest.ts` file only has
`import "@testing-library/jest-dom`

vite config file has `globals: true` and `environment: jsdom`.

Storybook also seems to add jsdom as a dev dependency, which wasn't required before adding Storybook to the project. I don't think it's the cause. Any idea why the coverage command seems to cause the test to stall forever and then fail?

### Additional information

I've excluded any stories-related files in `vite.config.ts`
Using node `v18.13.0`

These are dev dependencies:
"@storybook/addon-actions": "^7.6.10",
"@storybook/addon-essentials": "^7.6.10",
"@storybook/addon-interactions": "7.6.8",
"@storybook/blocks": "7.6.8",
"@storybook/react": "7.6.8",
"@storybook/react-vite": "7.6.8",
"@storybook/test": "7.6.8",
"eslint-plugin-storybook": "^0.6.15",
"jsdom"： “^23.2.0"，
"msw"： "^1.3.2"」
"msw-storybook-addon": "^1.10.0",

**Comment by valentinpalkovic** [comment]
@kasperpeulen and @yannbf may have some idea here.

**Comment by hfall** [comment]
I've changed `@testing-library/jest-dom` version to `^6.4.2` and jsdom to `^22.1.0`. It just works. I'll close this in a day or 2 if there's no alternative answers.
